R/nm-tables.R

```diff
@@ -88,7 +88,7 @@ nm_table_files <- function(.mod, .check_exists = TRUE) {
   out_dir <- get_output_dir(.mod, .check_exists = .check_exists)
 
   # get file names from table statements and construct paths
-  .f <- .l[names(.l) == "TABLE"] %>%
+  .f <- .l[grep("TAB", names(.l))] %>%
```

Comment:
I'd say make this `grep("^TAB", names(.l))` so that we don't snag any names that happen to have `TAB` in the middle somewhere. I don't know if that's even a possibility with NONMEM, but if they allow `$TAB` or `$TABLE` for specifying it then there shouldn't be any others that _start_ with `TAB`

Reply:
Yeah that's a good call - taken care of